src/Gui/PythonWrapper.cpp

```diff
@@ -75,24 +75,15 @@
 # ifdef HAVE_PYSIDE2
 #  define HAVE_PYSIDE
 
-// Since version 5.12 shiboken offers a method to get wrapper by class name (typeForTypeName)
-// This helps to avoid to include the PySide2 headers since MSVC has a compiler bug when
-// compiling together with std::bitset (https://bugreports.qt.io/browse/QTBUG-72073)
-
 // Include shiboken first to get the version
 #  include <shiboken.h>
 
 // Do not use SHIBOKEN_MICRO_VERSION; it might contain a dot
 #  define SHIBOKEN_FULL_VERSION QT_VERSION_CHECK(SHIBOKEN_MAJOR_VERSION, SHIBOKEN_MINOR_VERSION, 0)
-#  if (SHIBOKEN_FULL_VERSION >= QT_VERSION_CHECK(5, 12, 0))
-#   define HAVE_SHIBOKEN_TYPE_FOR_TYPENAME

```

Comment:
Don't we want to keep this? 5.15 is still more than 5.12

Reply:
(We want to keep the code, but remove the check)